packages/data-format/src/format/rnf_expense/README.md

```diff
@@ -0,0 +1,60 @@
+
+## meta
+
+_Information about the format of the json_
+
+| Name    | Type     | Need      | Comment              |
+| ------- | -------- | --------- | -------------------- |
+| format  | constant | Mandatory | value: "rnf_expense" |
+| version | constant | Mandatory | value: "0.1.0"       |
+
+## employeeInfo
+
+_Information about the employee_
+
+| Name       | Type                     | Need      | Comment            |
+| ---------- | ------------------------ | --------- | ------------------ |
+| email      | string _(format: email)_ | Mandatory | email              |
+| employeeId | string                   | Mandatory | id of the employee |
+| firstName  | string                   | Mandatory | first name         |
+| lastName   | string                   | Mandatory | last name          |
+
+## employerInfo
+
+_Information about the employer_
+
+| Name                | Type                     | Need      | Comment                                                                                                          |
+| ------------------- | ------------------------ | --------- | ---------------------------------------------------------------------------------------------------------------- |
+| email               | string _(format: email)_ | Mandatory | email                                                                                                            |
+| firstName           | string                   | Mandatory | first name                                                                                                       |
+| lastName            | string                   | Mandatory | last name                                                                                                        |
+| businessName        | string                   | Mandatory | business name                                                                                                    |
+| phone               | string                   | Optional  | phone                                                                                                            |
+| address             | object                   | Optional  | address formatted as [http://json-schema.org/address](http://json-schema.org/learn/examples/address.schema.json) |
+| companyRegistration | string                   | Optional  | company registration                                                                                             |
+| miscellaneous       | object                   | Optional  | Miscellaneous                                                                                                    |
+
+## expenseItem
+
+_List of the items of the expense_
+
+| Name        | Type      | Need      | Comment                                    |
+| ----------- | --------- | --------- | ------------------------------------------ |
+| name        | string    | Mandatory | name of the item                           |
```

Comment:
Same comment here, from a protocol point of view, we can have items with no names.